e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js

```diff
@@ -596,12 +596,7 @@ describe("scenarios > visualizations > line chart", () => {
 
       cy.wait("@dataset");
 
-      cy.findByTestId("filter-pill").should(
-        "have.text",
-        "Created At is Apr 1, 12:00 AM – Sep 1, 2022, 12:00 AM",
-      );
-
-      cy.get(".Visualization .dot").should("have.length", 6);
+      cy.findByTestId("filter-pill").should("contain.text", "Created At is");
```

Comment:
It would be nice to still assert that the date filter is correct because if drill provides invalid values I guess there will be `Created At is` and the test will pass

Reply:
Maybe querying chart axis ticks would be enough to ensure it is rendered

Reply:
It's not a test for a drill, it's a test for a brush filter 😄 anyhow, we need this test only to be sure that upcoming work about MLv2 migration of the related filters will still work.
Unfortunately, cypress doesn't provide deterministic results even if we pass correct coordinates for drag start and drag end - results vary between runs even locally

Reply:
In ECharts scope we had to reimplement brush drilling so it would be handy there as well. I think if you add findByText(*expected x-axis tick value*) it would cover the scenario slightly better because invalid filters would produce wrong query and won't render anything

Reply:
Agree, maybe we could check for some value in between the range in the chart that won't be flaky

Reply:
https://github.com/metabase/metabase/pull/37236/commits/20a8711f76784502c4664afbfc729f15f387bee8